integrations/phoneinfoga/compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+    phoneinfoga:
+      container_name: phoneinfoga
+      restart: unless-stopped
+      image: sundowndev/phoneinfoga:latest
+      command:
+        - "serve"
+      expose:
+        - "5000"
+      env_file:
+      - env_file_integrations
+      environment:
+      - GIN_MODE=release
```

Comment:
reference to this?

Reply:
this should run the GIN framework used in release mode. It works fine even without it.